examples/react-full-v1/renderer/+onRenderHtml.tsx

```diff
@@ -34,10 +34,13 @@ async function onRenderHtml(pageContext: PageContextServer) {
   return {
     documentHtml,
     // See https://vike.dev/stream#initial-data-after-stream-end
+    // temp(aurelien): interesting that this can be an async function. Can this be the case in
+    // onBeforeRender() as well? Or is this specific to onRenderHtml()?
```

Comment:
@brillout see inline question, thanks!

Reply:
It's only implemented for `onRenderHtml()`. FYI the user case is this: https://vike.dev/stream#initial-data-after-stream-end.

Reply:
Perfect, thanks